src/platform/forms-system/test/js/widgets/RadioWidget.unit.spec.jsx

```diff
@@ -136,4 +137,56 @@ describe('Schemaform <RadioWidget>', () => {
     expect(tree.everySubTree('input')[0].props['data-test']).to.equal('first');
     expect(tree.everySubTree('input')[1].props['data-test']).to.equal('second');
   });
+  it('should add custom and selected props', () => {
+    const onChange = sinon.spy();
+    const options = {
+      enumOptions: [
+        { label: 'Testing', value: '1' },
+        { label: 'Testing2', value: '2' },
+      ],
+      widgetProps: {
+        1: { 'data-test': 'first' },
+        2: { 'data-test': 'second' },
+      },
+      selectedProps: {
+        1: { 'data-selected': 'first' },
+        2: { 'data-selected': 'second' },
+      },
+    };
+    const tree = SkinDeep.shallowRender(
+      <RadioWidget value="1" onChange={onChange} options={options} />,
+    );
+    const inputs = tree.everySubTree('input');
+    expect(inputs[0].props['data-test']).to.equal('first');
+    expect(inputs[0].props['data-selected']).to.equal('first');
+
+    expect(inputs[1].props['data-test']).to.equal('second');
+    expect(inputs[1].props['data-selected']).to.be.undefined;
+  });
+  it('should add custom and selected props', () => {
```

Comment:
Is the only difference between this and the previous test the `value` being 2? If so, can that be communicated a bit more, or the tests restructured?

Reply:
I've updated the test to re-render with new values. Included changes in the `YesNoWidget` test as well.